### PR TITLE
Replace calls of "contains" with "in" operator

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Using "in" is shorter. Also this increases consistency because "in" is
already used in several places.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/604)

### CODE REVIEW

File: model/src/main/kotlin/Scope.kt [hunk 1]
```diff
@@ -59,12 +59,12 @@ data class Scope(
     /**
      * Returns whether the given package is contained as a (transitive) dependency in this scope.
      */
-    fun contains(pkg: Package) = contains(pkg.id)
+    operator fun contains(pkg: Package) = contains(pkg.id)
```
Comment (sschuberth) [inline model/src/main/kotlin/Scope.kt]: How is this change related?

Comment (mnonnenmacher) [reply model/src/main/kotlin/Scope.kt]: `operator` is required to be able to use `in` for this function.

Comment (sschuberth) [reply model/src/main/kotlin/Scope.kt]: Interesting. I didn't find this documented anywhere. [This](https://kotlinlang.org/docs/reference/operator-overloading.html#in) just says `a in b` is literally translated to `b.contains(a)`. But I double-checked that the `operator` is indeed required.
